src/qml/doc/translator-comments.md

```diff
@@ -0,0 +1,60 @@
+# Translator Comment Policy
+
+Translator comments are required for all user-facing strings. A PR introducing
+new user-facing strings must also submit the accompanying translator comment.
+To prevent this requirement from halting development, the PR author can specify
+that they would like the introduction of the comment to be picked up in a follow-up PR.
+
+## Formulating Translator Comments
+
+Translator comments only apply to strings that are marked for translation.
+For more information, see:
+[Internationalization and Localization with Qt Quick](https://doc.qt.io/qt-5/qtquick-internationalization.html)
+
+Below, lines 4 and 5 are examples of new user-facing strings marked for
+translation without translator comments:
+
+```c++
+1.  OptionButton {
```

Comment:
Why line numbers? My suggestion is to drop them and above say "Below, `text` and `description` values are examples of new user-facing ..."